conf/messages

```diff
@@ -20,12 +20,17 @@ transferAgreement.english=The records are all in English
 transferAgreement.digital=The records are all Digital
 transferAgreement.droAppraisalSelection=I confirm that the DRO has signed off on the appraisal and selection decision.
 transferAgreement.droSensitivity=I confirm that the DRO has signed off on the sensitivity review and that all records are OPEN i.e. no FOI exemptions apply to these records.
-upload.header=Upload
-upload.title=Upload
+upload.header=Upload Records
+upload.title=Upload Records
+upload.selectFolder=Select a folder
+upload.fileExtensionTitle=Which file extensions are not permitted?
+upload.fileExtensionSummary=Cannot upload the following file extensions: .zip, thumbs.db, .exe
+upload.continueLink=continue
```

Comment:
Should Continue have an upper case C? Sorry to keep nitpicking!

Reply:
Not at all! This is extremely helpful because you can point out mistakes I would overlook!